packages/env-tests/src/scaffold.ts

```diff
@@ -21,22 +36,24 @@ export async function fundAccount(
   })
   context.kit.connection.addAccount(root.privateKey)
 
-  const stableToken = await context.kit.contracts.getStableToken()
+  for (const stableToken of context.stableTokensToTest) {
```

Comment:
What do you think about parameterizing the running of the test instead of parameterizing the the context? 

I.e. it seems unnecessary to fund an account in let's say the attestation tests with cEUR. Instead, we could just parameterize `runTransferTest` with a `stableToken` parameter and pass that through to `fundAccount`?

Reply:
Makes sense. We set up a stabletoken parameterisation for the individual tests. Is it in line with what you had in mind?